source/_dashboards/gauge.markdown

```diff
@@ -150,23 +153,94 @@ segments:
   - from: 40
     color: '#43a047'
   - from: 60
-    color: '#ffa600'
+    color: '#4396a0'
   - from: 65
+    color: '#437ba0'
+  - from: 70
+    color: '#4362a0'
+```
+
+#### Outdoor temperature
+
+<p class='img'>
+<img src='/images/dashboards/gauge_segments_feelslike_temperature.png' alt='Screenshot of a gauge card with colored segments for temperature.'>
+Screenshot of a gauge card with colored segments for temperature.
+</p>
+
+```yaml
+type: gauge
+entity: sensor.feelslike_temperature
+needle: true
+min: -15
+max: 50
+segments:
+  - from: -15
+    color: '#8438e0'
+  - from: -5
+    color: '#4362a0'
+  - from: 2
+    color: '#4362a0'
+  - from: 9
+    color: '#4396a0'
+  - from: 16
+    color: '#43a047'
+  - from: 25
+    color: '#ffa600'
+  - from: 36
     color: '#db4437'
+  - from: 45
+    color: '#7e0023'
 ```
 
-CSS variables can be used (instead of CSS '#rrggbb') for default gauge segment colors:
+#### Air Quality Index (AQI)
+
+
+<p class='img'>
+<img src='/images/dashboards/gauge_segments_aqi.png' alt='Screenshot of a gauge card with colored segments for Air Quality Index.'>
+Screenshot of a gauge card with colored segments for Air Quality Index.
+</p>
+
+```
+type: gauge
+entity: sensor.u_s_air_quality_index
+min: 0
+max: 320
+name: AQI
+needle: true
+segments:
+  - from: 0
+    color: '#009966'
+  - from: 51
+    color: '#ffde33'
+  - from: 101
+    color: '#ff9933'
+  - from: 151
+    color: '#cc0033'
+  - from: 201
+    color: '#660099'
+  - from: 300
+    color: '#7e0023'
+```
+
+### CSS variables
+
+CSS variables can be used (instead of CSS '#rrggbb') for default gauge segment colors.
 
 - `var(--success-color)` for green color
 - `var(--warning-color)` for yellow color
 - `var(--error-color)` for red color
 - `var(--info-color)` for blue color
```

Comment:
Interesting, we should remove those, as those are actually not supported in theming.